src/axolotl/utils/data.py

```diff
@@ -637,6 +640,12 @@ def load_pretraining_dataset(path, tokenizer, max_tokens=2048, seed=42):
     encode = functools.partial(encode_pretraining, tokenizer, max_tokens)
     dataset = load_dataset(path, streaming=True, split="train")
     dataset = dataset.shuffle(seed=seed, buffer_size=10_000)
-    # TODO dynamically figure out which columns/features to remove
```

Comment:
can you leave this TODO please? some datasets have additional features that need to be dropped

Reply:
oh, nvm, I see now, map w/ input_columns just passes that so the rest aren't even present 👍